sdk/communication/AzureCommunicationChat/Source/Signaling/TrouterUtils.swift

```diff
@@ -0,0 +1,432 @@
+//
+//  TrouterUtils.swift
+//  AzureCommunicationSignaling
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import AzureCommunication
+import AzureCore
+import Foundation
+import TrouterClientIos
+
+func toEventPayload(request: TrouterRequest, chatEventId: ChatEventId) -> Any? {
+    do {
+        switch chatEventId {
+        case ChatEventId.chatMessageReceived:
+            return try toChatMessageReceivedEvent(request: request)
```

Comment:
These "toFoo" methods are not used idiomatically for Swift. It would make more sense for these to have initializers or static methods that work with the request:
```swift
// static method
ChatMessageReceivedEvent.from(request: request)

// convenience initializer
ChatMessageReceivedEvent(request: request)
```